src/core/selinux-access.c

```diff
@@ -163,6 +164,33 @@ static int access_init(sd_bus_error *error) {
         return 1;
 }
 
+int mac_selinux_check(
+                void *blob,
+                const char *path) {
+
+        _cleanup_free_ char *path_persistent = NULL;
+        struct mac_selinux_blob *t = NULL;
```

Comment:
Either drop this initalization or just assing 'blob' here directly (my preference would be for that).